frontend/windows/handle_zip.go

```diff
@@ -0,0 +1,168 @@
+package windows
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/Azure/dalec"
+	"github.com/Azure/dalec/frontend"
+	"github.com/moby/buildkit/client/llb"
+	"github.com/moby/buildkit/exporter/containerimage/image"
+	gwclient "github.com/moby/buildkit/frontend/gateway/client"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+)
+
+const (
+	workerImgRef    = "mcr.microsoft.com/mirror/docker/library/ubuntu:jammy"
+	outputDir       = "/tmp/output"
+	buildScriptName = "_build.sh"
+)
+
+var (
+	varCacheAptMount = llb.AddMount("/var/cache/apt", llb.Scratch(), llb.AsPersistentCacheDir("dalec-windows-var-cache-apt", llb.CacheMountLocked))
```

Comment:
For these to work, we also need to remove `/etc/apt/apt.conf.d/docker-clean` from the base image.
You should be able to acheive this by mounting over the file, like `llb.AddMount("/etc/apt/apt.conf.d/docker-clean", llb.Scratch())`

Reply:
- [ ] (Peter) Fix this

Reply:
done